src/ml_gym/persistency/io.py

```diff
@@ -24,6 +32,12 @@ def get_checkpoint_resource(self, grid_search_id: str, experiment_id: str,  chec
                                 checkpoint_resource: CheckpointResource):
         raise NotImplementedError
 
+    def post_checkpoint_resource_call(self, grid_search_id: str, experiment_id: str, payload: Dict):
+        raise NotImplementedError
+
+    def del_checkpoint_resource_call(self, grid_search_id: str, experiment_id: str, epoch: int):
```

Comment:
the function names were changed because there were same function names with different parameters and return type making it difficult to follow sometimes.
Reverted to name given.